FeedSX/screens/UniversalFeed/index.tsx

```diff
@@ -189,7 +293,68 @@ const UniversalFeed = () => {
   };
 
   return (
-    <View style={{height: '100%'}}>
+    <SafeAreaView style={{height: '100%'}}>
+      {/* header */}
+      <LMHeader heading="LikeMinds Sample App" />
```

Comment:
Make file `strings.ts` and export all the strings from there and use it here

Reply:
resolved